src/test/ui/hrtb/issue-94034.rs

```diff
@@ -0,0 +1,94 @@
+// known-bug
```

Comment:
I might take all of these and turn them into like... `known-ice` or something. Like automatically expect failure status 101, set backtrace=0, normalize all of the ICE notes, etc.

Reply:
That would be nice :)